package.json

```diff
@@ -28,11 +28,15 @@
   "main": "dist/rapidoc-min.js",
   "module": "dist/rapidoc-min.js",
   "dependencies": {
+    "@braintree/sanitize-url": "^4.0.0",
+    "crypto": "^1.0.1",
+    "jshint": "^2.11.0",
     "lit-element": "2.2.1",
     "lit-html": "1.0.0",
     "marked": "^0.8.0",
     "swagger-client": "^3.10.0",
-    "swagger2openapi": "^5.3.4"
+    "swagger2openapi": "^5.3.4",
+    "url-parse": "^1.4.7"
```

Comment:
I would like to keep our bundle size to minimum as we do not do any code splitting.  I dont think any of the packages added here is needed for our use-case